### ISSUE TITLE
Docker images for Chemotion>1.8.2 missing

### ISSUE THREAD

**Issue by behnle** [ComPlat/chemotion_ELN#1890]
I am unable to find any pre-built Docker images for current Chemotion releases.
Notably, the official "manual installation" instructions (https://www.chemotion.net/docs/eln/install_configure/manual_install) also link a docker-compose file (https://raw.githubusercontent.com/Chemotion/ChemCLI/main/payload/docker-compose.yml) which still requests Chemotion 1.8.2.
Is there any way of getting newer Docker images or do i have to build them on my own?

I am setting up a
- production

# Failure Information 

There are no Docker images for Chemotion 1.9.1 or 1.9.2 available. The newest images available from Dockerhub are 1.8.2 (https://hub.docker.com/r/ptrxyz/chemotion/tags)

## Context

I am 
- using Docker
- OS:  Rocky Linux 9.3
- ELN version 1.8.2, want upgrade to 1.9.2

## Steps to Reproduce

1. Check available images on Dockerhub

**Comment by harivyasi** [comment]
@behnle Thanks for opening the issue. Unfortunately, this time, our internal tests showed that versions tagged as 1.9.0 and 1.9.1 were not performing very well. Therefore, we decided not to release the built images publicly (i.e. on DockerHub). [Versions 1.9.2 and 1.9.3 were released](https://hub.docker.com/r/ptrxyz/chemotion/tags), with the last one being released yesterday. Thanks for your patience, we hope that you enjoy using them. :)
